### PR TITLE
[NAT] Get IP prefix from switch to replace hardcoded.

### PR DESCRIPTION
**Pull request by chaoskao**
# Description of PR
Get IP prefix from switch to replace hardcoded.

## Summary:
Fixes cannot run loopback and port_in_lag in same test case.

## Type of change
- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)
## What is the motivation for this PR?
Run NAT test case shows failed on port_in_lag.

## How did you do it?
Get IP prefix from switch to replace hardcoded.

## How did you verify/test it?
Run test cases and results show loopback and port_in_lag passed.

## Any platform specific information?
No

## Supported testbed topology if it's a new test case?
No

### CODE REVIEW

File: tests/nat/nat_helpers.py [hunk 1]
```diff
@@ -476,7 +476,10 @@ def setup_ptf_interfaces(testbed, ptfhost, duthost, setup_info, interface_type,
     if "dut_iface" in setup_info[interface_type]["vrf_conf"][key].keys():
         dut_iface = setup_info[interface_type]["vrf_conf"][key]["dut_iface"]
         pch_ip = setup_info["pch_ips"][dut_iface]
-        duthost.shell("sudo config interface ip remove {} {}/31".format(dut_iface, pch_ip))
+        pch_mask = setup_info["pch_masks"][dut_iface]
+        mask_prefix = IPAddress(pch_mask).netmask_bits()
```
Comment (vmorokhx) [inline tests/nat/nat_helpers.py:480]: You forgot to import: 
`from netaddr import IPAddress`
Or I miss something?

Comment (chaoskao) [reply tests/nat/nat_helpers.py:481]: thanks

**Comment by chaoskao** [comment]
retest please

**Comment by chaoskao** [comment]
@yxieca 
Can you please review

**Review by yxieca — APPROVED** [review]

**Comment by chaoskao** [comment]
@vmorokhx 
Can you please review

**Pull request merged**